.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js:
   - '10'
-  - '12.13.0' # Pinned due to a bug: moudle loading error (jest) 
+  - '12'
+cache:
```

Comment:
this is needed because the npm caching  mechanism in travis affects `@theforeman/test`